keyboards/handwired/d48/README.md

```diff
@@ -0,0 +1,76 @@
+# D48
+
+![Proton C based handwired 40% keyboard](https://i.imgur.com/2wCYuno.jpg)
+
+A Proton C based handwired 48 key keyboard with 2 rotary encoders, I2C OLED, WS2812 strip, buzzer & clock!
+
+- Keyboard Maintainer: Andrew Dunai
+- Hardware Supported: Proton C handwired
+
+Make example for this keyboard (after setting up your build environment):
+
+    make d48:anderson:dfu-util
+
+## Details
+
+- Proton C based handwired keyboard
+- 2x custom 1.25mm stainless steel plates
+- Kailh Choc White (clicky)
+- 2x rotary encoders
+- 0.91" 128x32 I<sup>2</sup>C OLED
+- Small buzzer mounted inside (still waiting for the AST1109MLTRQ boys)
+- WS2812 strip (14 LEDs)
+- DS1307 I<sup>2</sup>C module real-time clock
+
+Build process: [album](https://imgur.com/gallery/zZZGdDw)
+
+## Pinout
+
+![D48 pinout](https://imgur.com/LgHDQUU.jpg)
+
+## Challenges
+
+I'm very happy with the result, but at some point Proton C was driving me nuts.
+
+I did a lot of trial and error during assembly & programming.
+There were a lot of *yet* undocumented caveats,
+so I'll outline them here so that you guys can avoid the same issues I had.
+
+## I<sup>2</sup>C/OLED
+
+Most of the stuff worked out of the box, except me choosing the right pins for my OLED.
+
+On the Proton C pinout, there are 3 labels for I<sup>2</sup>C and for some reason
+there are 2 pairs of SDA/SCL for I<sup>2</sup>C<sup>1</sup> channel: `B8`/`B9` (rear left side) and `B6`/`B7` (rear right side).
+I'm not sure if this is a mistake or if I was doing something wrong. So initially I picked `B8`/`B9`
+which were not working. When I switched to B6/B7, things worked like a charm.
+Later I used B9 for matrix row. No issues so far.
+
+Oh, and by the way, while using `B8`/`B9`, keyboard was sometimes *swallowing* quick keypresses.
+I believe this was due to I(2)C timeouts (because incorrect pins were used for OLED).
+
+## Buzzer
+
+It turns out once you switch on `AUDIO_ENABLE`, you cannot use A4 & A5 because they interfere with the buzzer.
+My guess is that buzzer uses DAC channels (not sure why both).
```

Comment:
Correct.  And it's hard wired for both, right now.  It's something that should be fixed in the future.